package.json

```diff
@@ -21,7 +21,9 @@
 		"fastify-cors": "^6.0.2",
 		"fastify-multipart": "^5.1.0",
 		"fastify-rate-limit": "^5.7.0",
-		"sqlite3": "^5.0.2"
+		"puppeteer": "^13.3.1",
```

Comment:
We really should find a way to use an API instead of running a headless chrome instance. Both due to the massive overhead as well as any changes to page layout having the possibility of breaking the auth, compared to APIs which tend to be more stable.